admin/app/pagepresser/InteractiveHtmlCleaner.scala

```diff
@@ -0,0 +1,100 @@
+package pagepresser
+
+import com.netaporter.uri.Uri._
+import org.jsoup.Jsoup
+import org.jsoup.nodes.{Element, Document}
+import scala.collection.JavaConversions._
+import scala.io.Source
+
+object InteractiveHtmlCleaner extends HtmlCleaner with implicits.WSRequests {
+
+  override def canClean(document: Document): Boolean = {
+    document.getElementById("interactive-content") != null
+  }
+
+  override def clean(document: Document): Document = {
+    universalClean(document)
+    removeScripts(document)
+    createSimplePageTracking(document)
+    removeByTagName(document, "noscript")
+  }
+
+  override def extractOmnitureParams(document: Document) = {
+    val omnitureNoScript = if(document.getElementById("omnitureNoScript") != null) {
+      document.getElementById("omnitureNoScript")
+    } else {
+      // Dear reviewer, this horrible code is here because I've found examples of interactives that have nested iframes.
+      // Yes, honestly. Nested iframes. Which Jsoup struggles to parse properly.
+      // Anyway, the omnitureNoScript element we're seeking may exist at some nested level and so this is an attempt at
+      // working arounf the shortcomings of Jsoup's parser. It's horrible and I'm sorry. But I need to get on with
+      // pressing these damned things. If you know a better way, please say.
```

Comment:
I think three attempts it more than enough. We should just pass the interactives in small groups. Hopefully it is just a few that we can manually fix up.